# request
sensor_msgs/JointState joints_state
---
# response
geometry_msgs/Pose pose
